charts/astronomer/templates/houston/houston-configmap.yaml

```diff
@@ -345,7 +345,7 @@ data:
     elasticsearch:
       enabled: true
       client:
-        host: {{ printf "%s-external-es-proxy:9201" .Release.Name }}
+        node: {{ printf "http://%s-external-es-proxy:9201" .Release.Name }}
```

Comment:
Is using `http://` and not `https://` a problem here? Seems like it would be.

Reply:
Oh wait this is the external es url? 
The motivation was to keep `http://` for internal es urls

Reply:
ok, as long as it's needed. would be great if internal was also https though, but that's something for another issue.

Reply:
Added `https://` for custom logging config and `http:// `for internal for now